### PR TITLE
Add NEWS.md to data.atmosphere

### PR DESCRIPTION
**Pull request by infotroph**
Tried to summarize the changes I could see in Git history, but a lot of these are components I don't know well -- please correct my descriptions where I got it wrong, and add developments I've missed

Help wanted especially on the entries with question marks in them.

### CODE REVIEW

File: modules/data.atmosphere/NEWS.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* `download.MERRA()` now respects `overwrite = FALSE` [@meetagrawal09, #3169]
+* `download.MERRA()` can now find data from September of 2020 [@KristinaRiemer, #2888]
+* `download.Fluxnet2015()` now gives a more useful message when passed a malformed URL [@meetagrawal, #3130]
+
+## Added
+
+* Newly supported met data sources:
+	- ERA5 via `ERA5_met_process()` [@DongchenZ, #2886]
+	- NOAA GEFS via `download_NOAA_GEFS_EFI()` [@helge22a, #3174]
+		(replaces NOAA_GEFS_downscale?)
```
Comment (mdietze) [inline modules/data.atmosphere/NEWS.md:17]: No, doesn't replace. The EFI version can only grab files for the subset of sites on data.ecoforecast.org, which at the moment is just NEON. We still need to maintain the original GEFS download functions for all other sites

File: modules/data.atmosphere/NEWS.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	- ICOS?
+	- MERRA?
+* half-hourly downscaling via `downscale_*_to_half_hrly()`?
```
Comment (mdietze) [inline modules/data.atmosphere/NEWS.md:20]: I'm not sure about these 3. What PRs were they associated with?

Comment (infotroph) [reply modules/data.atmosphere/NEWS.md:20]: Hmm, looks like all of these were added prior to the 1.7.2 release:
* ICOS: https://github.com/PecanProject/pecan/pull/2813
* MERRA: https://github.com/PecanProject/pecan/pull/2725
* half-hourly: https://github.com/PecanProject/pecan/pull/2800

I'll remove and let people find them in the big changelog.

File: modules/data.atmosphere/NEWS.md (hunk 1 above)
Comment (infotroph) [inline modules/data.atmosphere/NEWS.md:17]:
```suggestion
	- NOAA GEFS data hosted by the Ecological Forecasting Initiative, via `download_NOAA_GEFS_EFI()` [@helge22a, #3174] (For GEFS data directly from NOAA, continue to use `download.NOAA_GEFS`)
```

File: modules/data.atmosphere/NEWS.md (hunk 2 above)
Comment (infotroph) [inline modules/data.atmosphere/NEWS.md:20]:
```suggestion
```

**Review by mdietze — APPROVED** [review]

**Pull request merged**